### ISSUE TITLE
Error related to parameterization changes

### ISSUE THREAD

**Issue by lochotzke** [eerimoq/asn1tools#51]
Since version 0.143.0 I get the following error compiling [all.asn](https://github.com/eerimoq/asn1tools/files/2768982/all.asn.txt)
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/asn1tools/compiler.py", line 354, in compile_files
    numeric_enums)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/compiler.py", line 284, in compile_dict
    numeric_enums),
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/ber.py", line 1643, in compile_dict
    return Compiler(specification, numeric_enums).process()
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 186, in process
    self.pre_process()
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 234, in pre_process
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 472, in pre_process_parameterization_step_1
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 544, in pre_process_parameterization_step_1_type
    del type_descriptor['parameters']
KeyError: 'parameters'
```
I created [all.asn](https://github.com/eerimoq/asn1tools/files/2768982/all.asn.txt) from files found in [ts_10309603v010401p0.zip](https://github.com/eerimoq/asn1tools/files/2768991/ts_10309603v010401p0.zip) originally from [here](https://www.etsi.org/deliver/etsi_ts/103000_103099/10309603/01.04.01_60/ts_10309603v010401p0.zip) with `cat EtsiTs103097Module.asn 1609Dot2/* TS102921/* > all.asn`. I also had to uncomment two imports in EtsiTs102941MessagesCA.asn.
The files can also be found in the [ETSI Git repository](https://forge.etsi.org/gitlab/ITS/ITS_ASN1).

**Comment by lochotzke** [comment]
I now cloned the [ETSI repository](https://forge.etsi.org/gitlab/ITS/ITS_ASN1.git) and created all.asn with `cat SEC_TS103097/* PKI_TS102941/* > all.asn`.

`compile_files` leads to:
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/asn1tools/compiler.py", line 354, in compile_files
    numeric_enums)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/compiler.py", line 284, in compile_dict
    numeric_enums),
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/ber.py", line 1643, in compile_dict
    return Compiler(specification, numeric_enums).process()
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 186, in process
    self.pre_process()
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 234, in pre_process
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 472, in pre_process_parameterization_step_1
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 540, in pre_process_parameterization_step_1_type
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 540, in pre_process_parameterization_step_1_type
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 507, in pre_process_parameterization_step_1_type
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 962, in lookup_type_descriptor
    return self.lookup_in_modules('types', 'type', type_name, module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 959, in lookup_in_modules
    module_name))
asn1tools.errors.CompileError: Type 'EtsiTs103097Data-Encrypted' not found in module 'EtsiTs102941MessagesCa'.
```
This can be fixed by uncommenting line 14 and 16 and re-creating all.asn.

With the fixed imports [all.asn](https://github.com/eerimoq/asn1tools/files/2772845/all.asn.txt) I get:
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/asn1tools/compiler.py", line 354, in compile_files
    numeric_enums)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/compiler.py", line 284, in compile_dict
    numeric_enums),
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/ber.py", line 1643, in compile_dict
    return Compiler(specification, numeric_enums).process()
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 186, in process
    self.pre_process()
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 234, in pre_process
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 472, in pre_process_parameterization_step_1
    module_name)
  File "/usr/local/lib/python3.7/site-packages/asn1tools/codecs/compiler.py", line 544, in pre_process_parameterization_step_1_type
    del type_descriptor['parameters']
KeyError: 'parameters'
```

**Comment by eerimoq** [comment]
Fixed in 0.145.2, available on PyPI.
